smart-contracts/contracts/mixins/MixinApproval.sol

```diff
@@ -40,7 +38,7 @@ contract MixinApproval is
     require(
       isKeyOwner(_tokenId, msg.sender) ||
         _isApproved(_tokenId, msg.sender) ||
-        isApprovedForAll(ownerOf[_tokenId], msg.sender),
+        isApprovedForAll(_ownerOf[_tokenId], msg.sender),
```

Comment:
Hmm if we are switching to the ZOS pattern than the data would be private and we use the public function instead.

Reply:
They also have a require statement in the ownerOf implementation... not sure if that we should too though

Reply:
Yeah, I can review through and try to use the public function more consistently. As for the require, we do have it, but it's in the `isKey` modifier which is used in our `ownerOf` implementation.

Reply:
k, I've now used the public function everywhere I can, and made the data private. The only exceptions are for write operations.

Reply:
didn't mean to complicate things :)  i don't feel strongly either way if you want to revert to what you had